deploy/1-deploy-oracles.ts

```diff
@@ -123,4 +122,4 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
   });
 };
 
-export default func;
+module.exports.tags = ["deploy"];
```

Comment:
same here? Is there a problem with hardhat deploy plugin?

Reply:
I have reverted back to use the DeployFunction interface. I have added tags to selectively run scripts.